### PR TITLE
PMM-7326 log level for MySQL, MongoDB, PostgreSQL, ProxySQL exporters

### PR DESCRIPTION
**Pull request by YaroslavPodorvanov**
PMM-7326

Build: SUBMODULES-2471

- [ ] https://github.com/percona/pmm/pull/874
- [ ] https://github.com/percona/pmm-admin/pull/202
- [ ] https://github.com/percona/pmm-managed/pull/1087

### CODE REVIEW

**Comment by BupycHuk** [comment]
don't forget about inventory API.

**Comment by YaroslavPodorvanov** [comment]
> don't forget about inventory API.

@BupycHuk add "log level" to inventory agents list

**Comment by YaroslavPodorvanov** [comment]
> thanks, could you also add log level to all `AddXXXExporterRequest` and `ChangeXXXExporterRequest` requests in agents.proto. for Change `ChangeCommonAgentParams` looks like a best place to put.

@BupycHuk i can do it in new separate issue, because PMM-7326 describe only CLI acceptance criteria

**Comment by YaroslavPodorvanov** [comment]
@ritbl, @BupycHuk, @qwest812 can you review?

**Review by BupycHuk — APPROVED** [review]
Please create task or subtask for that, we have `pmm-admin inventory` commands which is a part of CLI and it uses API endpoints I mentioned here.

**Comment by YaroslavPodorvanov** [comment]
> Please create task or subtask for that

Did raw now https://jira.percona.com/browse/PMM-9953

File: api/managementpb/log_level.proto [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+option go_package = "api/managementpb;managementpb";
+
+// Log level for exporters
+enum LogLevel {
+  warn = 0;
+  debug = 1;
+  info = 2;
+  error = 3;
+  fatal = 4;
+}
```
Comment (artemgavrilov) [inline api/managementpb/log_level.proto:14]: We breaking backward compatibility with such changes. Now you **should** specify log level. I think we need some `default = 0` value here. 

@BupycHuk what do you think?

Comment (BupycHuk) [reply api/managementpb/log_level.proto:14]: Agree, for better consistency we can call it auto like for metrics mode

Comment (YaroslavPodorvanov) [reply api/managementpb/log_level.proto:14]: In issue https://jira.percona.com/browse/PMM-7326 :
```
while the default should be `–log–level=WARN`
```
so default = 0 = warn

Comment (YaroslavPodorvanov) [reply api/managementpb/log_level.proto:14]: > Agree, for better consistency we can call it auto like for metrics mode

Like metrics mode:
```proto
enum LogLevel {
  default = 0;
  error = 1;
  fatal = 2;
  warn = 3;
  info = 4;
  debug = 5;
}
```

```go
// PUSH or AUTO variant enables pushMode for the agent.
func isPushMode(variant managementpb.MetricsMode) bool {
	return variant == managementpb.MetricsMode_PUSH || variant == managementpb.MetricsMode_AUTO
}
```

Proposal solution:
```
func specifyLogLevel(variant inventorypb.LogLevel) string {
	if variant == inventorypb.LogLevel_default {
		return ""
	}
	
	return variant.String()
}
```
```go
		row, err := models.CreateAgent(tx.Querier, models.MongoDBExporterType, &models.CreateAgentParams{
			PMMAgentID:        req.PmmAgentId,
			ServiceID:         service.ServiceID,
			Username:          req.Username,
			Password:          req.Password,
			AgentPassword:     req.AgentPassword,
			TLS:               req.Tls,
			TLSSkipVerify:     req.TlsSkipVerify,
			MongoDBOptions:    mongoDBOptions,
			PushMetrics:       isPushMode(req.MetricsMode),
			DisableCollectors: req.DisableCollectors,
			LogLevel:          specifyLogLevel(req.LogLevel),
		})
```

Comment (BupycHuk) [reply api/managementpb/log_level.proto:14]: yeah, with options in comments.

Comment (YaroslavPodorvanov) [reply api/managementpb/log_level.proto:14]: > yeah, with options in comments.

done

Comment (BupycHuk) [reply api/managementpb/log_level.proto:14]: let's remove `log_level.*` from managementpb

**Review by artemgavrilov — DISMISSED** [review]

**Review by ritbl — DISMISSED** [review]
LGTM

**Review by YaroslavPodorvanov — COMMENTED** [review]
self review

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**